cluster/service/cluster.go

```diff
@@ -0,0 +1,153 @@
+package service
+
+import (
+	"context"
+	"github.com/fabric8-services/fabric8-cluster/application/service"
+	"github.com/fabric8-services/fabric8-cluster/application/service/base"
+	servicectx "github.com/fabric8-services/fabric8-cluster/application/service/context"
+	"github.com/fabric8-services/fabric8-cluster/cluster/repository"
+	"github.com/fabric8-services/fabric8-cluster/configuration"
+	"github.com/fabric8-services/fabric8-common/httpsupport"
+	"github.com/fsnotify/fsnotify"
+	log "github.com/sirupsen/logrus"
```

Comment:
We use "github.com/sirupsen/logrus" log in configuration to solve the cycle dependency between log and configuration. But here you can use ""github.com/fabric8-services/fabric8-common/log".

Reply:
done in 47923fb